src/core/ext/xds/xds_client_stats.h

```diff
@@ -89,9 +87,7 @@ class XdsLocalityName final : public RefCounted<XdsLocalityName> {
   const std::string& region() const { return region_; }
   const std::string& zone() const { return zone_; }
   const std::string& sub_zone() const { return sub_zone_; }
-  std::shared_ptr<std::map<std::string, std::string>> locality_labels() const {
-    return locality_labels_;
-  }
+  RefCountedStringValue locality() const { return locality_; }
```

Comment:
That means we'll create a new RefCountedStringValue each time we invoke `AsHumanReadableString()` which is unnecessary but it probably also doesn't matter much since this is on the control plane

Reply:
Yeah, that makes a lot more sense.